### ISSUE TITLE
Vesicoureteral reflux grade III

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6827]
**Preferred term label:**
Vesicoureteral reflux grade III

**Synonyms**
VUR grade III, VUR III

**Definition (free text, please give PubMed ID)**
Vesicoureteral reflux with dilatation of the collecting system.
PMID: 3975102, 18322164

**Parent term (use hpo.jax.org/app)**
High-grade vesicoureteral reflux (newly proposed term)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
